### ISSUE TITLE
[Mobile] Refreshing page block continuation of activities on PDP

### ISSUE THREAD

**Issue by ibugaj** [DivanteLtd/vue-storefront#2484]
## Current behavior
Refreshing page block continuation of activities on PDP - functions don't work (changing size, color, add to favorite)

## Expected behavior
Refreshing page doesn't block continuation of activities on PDP
## Steps to reproduce the issue
1. Open any PDP on mobile
2. Refresh page
3. Try to change color or size, add to favorite, add to cart

## Environment details
https://test.storefrontcloud.io/

mobile phone

**Comment by pkarw** [comment]
@patzick is it still a case?

**Comment by pkarw** [comment]
Can't reproduce this issue anymore please do verify @ibugaj

**Comment by ibugaj** [comment]
@pkarw I still have this issue on my mobile. I use Chrome 72.0.3626.76 on Android 7.0

**Comment by pkarw** [comment]
@ibugaj can you clear storage on this mobile device and try once more?
@patzick can you check this one?

**Comment by ibugaj** [comment]
Yes @pkarw , I started from clear storage, but I have a little problem with this - it still keeps on waiting..:
![image](https://user-images.githubusercontent.com/47220013/53658980-8f468f00-3c5a-11e9-90aa-11498ad96dc3.png)
So I cleared history manually only. 
But earlier I checked this issue also on another phone and there also issue was

**Comment by ArturDivante** [comment]
I just investigated this issue and I can confirm - the problem exists.  

As far as I could check, it occurs only on **Chrome / Android** environment. Tested on many combinations, both in real devices and browserstack. 

Other browsers, as well as Chrome working on iOS seem to be unaffected.

**Comment by pkarw** [comment]
Can’t reproduce this error anymore

**Comment by ArturDivante** [comment]
@pkarw 

Unfortunately, bug still exist. As I mentioned before, it occurs on chrome browser working in android.

I just recreated it on all our test environments. Used my own phone, all physical devices we have on test stand, as well as browserstack - its universal. I cleared device cache before every test. 

Problem is NOT reproducable in mobive view of PC browser.

After refreshing product page:
Picture is replaced by placeholder
Clicking on filters (color, size) has no effect
Clicking 'addd to cart' displays cliking animation, but wont add product to cart
Adding to wishlist is not working

![refreshing PDP](https://user-images.githubusercontent.com/47415585/55711919-060b4080-59ee-11e9-8774-a84e2e9a4fea.gif)
